### PR TITLE
ux/fix(slack): move snooze mfa to new mfa, add emoji to case message,…

### PR DESCRIPTION
**Pull request by ghost**
… and fix genai prompt bug

Screenshot 2024-09-24 at 2 18 50 PM

### CODE REVIEW

File: src/dispatch/plugins/dispatch_slack/case/messages.py [hunk 1]
```diff
@@ -345,11 +347,13 @@ def create_genai_signal_summary(
     genai_plugin = plugin_service.get_active_instance(
         db_session=db_session, project_id=case.project.id, plugin_type="artificial-intelligence"
     )
-    if not genai_plugin:
+
+    print(f"{signal_instance.signal.__dict__}")
```
Comment (mvilanova) [inline src/dispatch/plugins/dispatch_slack/case/messages.py:351]:
```suggestion
```

**Review by mvilanova — CHANGES_REQUESTED** [review]

File: src/dispatch/plugins/dispatch_slack/case/interactive.py [hunk 2]
```diff
@@ -1735,7 +1744,7 @@ def report_issue(
         Context(
             elements=[
                 MarkdownText(
-                    text="Cases are meant to triage events that do not raise to the level of incidents, but can be escalated to incidents if necessary. If you suspect a security issue and need help, please fill out this form to the best of your abilities.."
+                    text="Cases are meant to triage events that do not raise to the level of incidents, but can be escalated to incidents if necessary."
```
Comment (mvilanova) [inline src/dispatch/plugins/dispatch_slack/case/interactive.py:1747]:
```suggestion
                    text="Cases are meant for triaging events that do not raise to the level of incidents, but can be escalated to incidents if necessary."
```

File: src/dispatch/plugins/dispatch_slack/case/messages.py [hunk 3]
```diff
@@ -345,11 +347,13 @@ def create_genai_signal_summary(
...
+    if not genai_plugin or not signal_instance.signal.genai_prompt:
         return signal_metadata_blocks
```
Comment (mvilanova) [inline src/dispatch/plugins/dispatch_slack/case/messages.py:353]: Should we separate the checks and log a warning when `genai_plugin` exists, but we don't have a prompt?

**Comment by mvilanova** [comment]
The icons in the buttons look nice!

File: src/dispatch/plugins/dispatch_slack/case/messages.py (hunk 1 above)
Comment (wssheldon) [reply src/dispatch/plugins/dispatch_slack/case/messages.py]: Yikes, thanks for catching!

File: src/dispatch/plugins/dispatch_slack/case/messages.py (hunk 3 above)
Comment (wssheldon) [reply src/dispatch/plugins/dispatch_slack/case/messages.py:359]: Good idea, done

**Review by mvilanova — APPROVED** [review]

File: src/dispatch/plugins/dispatch_slack/case/interactive.py [hunk 4]
```diff
@@ -860,8 +859,7 @@ def _create_snooze_filter(
         db_session.commit()
 
     # Check if last_mfa_time was within the last hour
-    last_hour = datetime.now() - timedelta(hours=1)
-    if (user.last_mfa_time and user.last_mfa_time > last_hour) or mfa_enabled is False:
+    if mfa_enabled is False:
```
Comment (metroid-samus) [inline src/dispatch/plugins/dispatch_slack/case/interactive.py:862]:
```suggestion
    if not mfa_enabled:
````

**Pull request merged**